pkg/risk/riskcontrol/position.go

```diff
@@ -45,27 +72,16 @@ func NewPositionRiskControl(orderExecutor bbgo.OrderExecutorExtended, hardLimit,
 		}
 
 		log.Infof("RiskControl: position limit exceeded, submitting order to reduce position: %+v", submitOrder)
-		createdOrders, err := orderExecutor.SubmitOrders(context.Background(), submitOrder)
+		createdOrders, err := p.orderExecutor.SubmitOrders(ctx, submitOrder)
```

Comment:
need to be careful here, the trade collector might get stuck

would better to run a back-test to test this change~

Reply:
I will run a back-test later

Reply:
the trade collector didn't get stuck